nginx.conf

```diff
@@ -190,7 +190,7 @@ server {
   rewrite ^/academy/web-scraping-for-beginners/crawling/processing-data$ /academy/web-scraping-for-beginners/crawling/exporting-data permanent;
   rewrite ^/academy/web-scraping-for-beginners/crawling/recap-collection-basics$ /academy/web-scraping-for-beginners/crawling/recap-extraction-basics permanent;
   rewrite ^/academy/php/using-apify-scraper-with-php$ /academy/php/use-apify-from-php permanent;
-  rewrite ^/api/?$ /api/v2 redirect;
```

Comment:
This is how we agreed with @B4nan how to redirect - we can rewrite original /v2. Also we want to redirect frome /api/client